controllers/comments.controllers.js

```diff
@@ -25,3 +26,13 @@ exports.postCommentByArticle = async (req, res, next) => {
     next(err);
   }
 };
+
+exports.deleteCommentById = async (req, res, next) => {
+  const {comment_id} = req.params
+  try {
+    await removeCommentById(comment_id)
+    res.status(204).send()
+  } catch (err) {
+    next(err)
```

Comment:
Here's some syntax shorthand you can take advantage of here:
- When sending nothing back,  you can use the method `.sendStatus(204)` 
- When using a catch block that only invokes next with the error, you can use `.catch(next)` and express will invoke next with the error if it goes to the catch